debian/build_virtualenv

```diff
@@ -37,7 +37,12 @@ python3 -m venv "$TEMP_VENV"
 source "$TEMP_VENV/bin/activate"
 pip install -U pip
 pip install poetry==1.2.0b1
-poetry export --extras all --extras test -o exported_requirements.txt
+poetry export \
+    --extras all \
+    --extras test \
+    --extras systemd \
+    --extras cache_memory \
```

Comment:
I can't help but wonder if this is the right fix, or if we should go back to having `all` include, well, all the extras. (Or, most of them.)

(this used to be included via [this code](https://matrix.org/blog/2019/11/09/avoiding-unwelcome-visitors-on-private-matrix-servers) and [this code](https://github.com/matrix-org/synapse/blob/v1.56.0/setup.py#L88-L91)).

Reply:
That is a good point. As it stands, the docker images also don't look to include `pympler`: https://github.com/matrix-org/synapse/blob/e5a76ec00b443c6e15ac8de2838a5c60a47a67ec/docker/Dockerfile#L64.

Reply:
I'm assuming that dependencies for an experimental config option (`cache.track_memory_usage`) are in fact something that we'd like to include in Debian and Docker images, as they would allow people using those distribution methods to actually test the option.

In that case, including them in `all` sounds sensible.

Reply:
sgtm

Reply:
> I can't help but wonder if this is the right fix, or if we should go back to having `all` include, well, all the extras. (Or, most of them.)
> 
> (this used to be included via [this code](https://github.com/matrix-org/synapse/blob/v1.56.0/synapse/python_dependencies.py?rgh-link-date=2022-05-05T11%3A21%3A21Z#L121-L127) and [this code](https://github.com/matrix-org/synapse/blob/v1.56.0/setup.py?rgh-link-date=2022-05-05T11%3A21%3A21Z#L88-L91)).

By my reading, the first link explicitly excludes `systemd` from `all`. I can't remember why I didn't include pympler; maybe just an oversight.